data/transactions/logic/assembler.go

```diff
@@ -1531,6 +1545,16 @@ func (ops *OpStream) trackStack(args StackTypes, returns StackTypes, instruction
 	}
 }
 
+// splitTokens breaks tokens into two slices at the first semicolon.
+func splitTokens(tokens []string) (current, rest []string) {
```

Comment:
```suggestion
func splitTokens(tokens []string) ([]string, []string) {
```more
I defer to others with more mastery of golang best practices, but unused named return variables strike me as error-prone.

Reply:
On the contrary, I think the _best_ use of named returned values is to disambiguate the meaning of return values, especially when they have the same type, so their meaning is less obvious.

Effective Go says:
> The names are not mandatory but they can make code shorter and clearer: they're documentation. If we name the results of nextInt it becomes obvious which returned int is which.